src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -213,6 +222,25 @@ public function get_all_registered( $outside_init_only = false ) {
 		);
 		$hooked_blocks = get_hooked_blocks();
 		foreach ( $patterns as $index => $pattern ) {
+			if ( ! isset( $pattern['name'] ) ) {
+				unset( $patterns[ $index ] );
+				if ( $outside_init_only ) {
+					unset( $this->registered_patterns_outside_init[ $index ] );
+				} else {
+					unset( $this->registered_patterns[ $index ] );
+				}
+				continue;
+			}
+			if ( ! isset( $pattern['content'] ) && isset( $pattern['file_path'] ) ) {
+				ob_start();
+				include $pattern['file_path'];
+				$pattern['content'] = ob_get_clean();
+				if ( $outside_init_only ) {
+					$this->registered_patterns_outside_init[ $index ]['content'] = $pattern['content'];
+				} else {
+					$this->registered_patterns[ $index ]['content'] = $pattern['content'];
+				}
+			}
```

Comment:
Similar to my comment in `get_registered()`, if we moved the file reading to `prepare_content()`, we could avoid duplicating this code. Alternately, we could put that in a separate private method so you can update the registered object prior to running it through `prepare_content()` to avoid running these file reads more than once (although I suspect the performance hit of doing so would be small, given the file contents should be in opcode cache).